flocker/acceptance/test_api.py

```diff
@@ -84,6 +85,20 @@ def base_url(self):
             self.control_node.address, REST_API_PORT
         )
 
+    def _json_content(self, result, response_code):
```

Comment:
:question: And perhaps this could be useful elsewhere, so make it a module level function.